test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Rendering/HtmlHelperLabelExtensionsTest.cs

```diff
@@ -155,20 +278,24 @@ public void LabelHelpers_DisplayDisplayName_IfNonNull(string displayName)
             Assert.Equal("<label for=\"\">HtmlEncode[[" + displayName + "]]</label>", HtmlContentUtilities.HtmlContentToString(labelForModelResult));
         }
 
+        // Prior to aspnet/Mvc#6638 fix, helpers generated nothing with this setup.
+        // Following test mimics use of an identity expression in an editor template if invoked for a property. See
+        // also LabelHelpers_ReturnExpectedElementForModel_IfDisplayNameEmptyAndNotTopLevel().
         [Fact]
-        public void LabelHelpers_ReturnEmptyForProperty_IfDisplayNameEmpty()
+        public void LabelHelpers_ReturnExpectedElementForProperty_IfDisplayNameEmptyAndNotTopLevel()
```

Comment:
Test previously returned empty because the setup was incorrect. Used neither `provider` nor `modelExplorer`.